llvm-bazel/terminfo_system.BUILD

```diff
@@ -0,0 +1,15 @@
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Wrapper library for some system terminfo. Using this only works if the
+# toolchain already has the relevant library search paths configured. It also
+# sets the relevant LLVM `#define`s to enoble using terminfo.
+cc_library(
+    name = "terminfo",
+    defines = ["LLVM_ENABLE_TERMINFO=1"],
+    # Note that we will replace the exact library here with the name detected
+    # on the system.
```

Comment:
We will? I think I maybe don't understand this sentence. We're replacing the linkopts with the ones detected on the system and I guess that's basically the name of the terminfo, but we're not replacing the library...

Reply:
Sorry, yes, stale comment. Updated.